examples/plotting/plotting_blank_map.py

```diff
@@ -0,0 +1,54 @@
+  
+"""
+==============================================
+Plot Positions on a blank Map
+==============================================
+This example shows how to plot random positions on a blank map
+"""
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import sunpy.map
+from sunpy.coordinates import frames
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.io import fits
+
+################################################################################
+# Creating blank map data with zeros
+data = np.zeros((1000, 1000))
+
+# Define coordinates and frame of reference and make the header using sunpy.map.make_fitswcs_header
+skycoord = SkyCoord(0*u.arcsec, 0*u.arcsec, obstime = '2013-10-28', observer = 'earth', frame = frames.Helioprojective)
+header = sunpy.map.make_fitswcs_header(data, skycoord)
+
+# Use the sunpy.map.Map to make the blank map
+blank_map = sunpy.map.Map(data, header)
+
+################################################################################
+
+# Initialize the plot and add the map to it
+fig = plt.figure()
+ax = plt.subplot(projection=blank_map)
+im = blank_map.plot()
```

Comment:
Do we need to do this?

Reply:
this defines the plot right?

Reply:
you dont need to `im=` you can just do `blank.plot` as you are not using a colorbar